src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -111,13 +117,33 @@ public static File renderTileMetricsFileFromBasecallingDirectory(final File illu
                 final int phasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PHASING_BASE);
                 final int prePhasingCode = IlluminaMetricsCode.getPhasingCode(descriptorIndex, IlluminaMetricsCode.PREPHASING_BASE);
 
-                if (!(codeMetricsMap.containsKey(phasingCode) && codeMetricsMap.containsKey(prePhasingCode))) {
-                    throw new PicardException("Don't have both phasing and prephasing values for tile");
+                float phasingValue = 0;
+                float prePhasingValue = 0;
+
+                // If both the phasing and pre-phasing data are missing, then likely something went wrong when imaging
+                // this tile, for example a grain of sand disrupting the path of light to the sensor.  If only one of them
+                // is missing, then likely the data is corrupt.
```

Comment:
if data is corrupt you should throw an exception...but you modified the old if statement to be stricter, and now the case that **only** one of the codes is missing is ignored...this case should still throw.